core/src/main/java/io/lionweb/lioncore/java/serialization/data/SerializedClassifierInstance.java

```diff
@@ -11,6 +11,7 @@ public class SerializedClassifierInstance {
   private String id;
```

Comment:
What's the difference to a `SerializedNode`?

Reply:
In my understanding we have annotation instances and nodes, which are similar but not the same. So the common ancestor (grouping the similarities) is called Classifier Instance. Also when serializing them they are very similar but not the same. For example, an annotation instance has no concept, it has an annotation instead. It has no parent but an annotated node instead (see previous comment). For this reason also on the serialization level I find useful to distinguish them.